### ISSUE TITLE
"Unit" is missing from HELP after #414

### ISSUE THREAD

**Issue by or-shachar** [prometheus/cloudwatch_exporter#422]
Minor regression that was probably caused by my #414. The unit is missing from the HELP:
### before:
```
# HELP aws_sqs_sent_message_size_average CloudWatch metric AWS/SQS SentMessageSize Dimensions: [QueueName] Statistic: Average Unit: Bytes
```

### after
```
# HELP aws_sqs_sent_message_size_average CloudWatch metric AWS/SQS SentMessageSize Dimensions: [QueueName] Statistic: Average Unit: null
```

I can probably fix it in case of `use_get_metric_data` is set to `false` but in case it is set to true - we don't really have a way to automatically set the unit as this is not part of the [GetMetricData response signature](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_GetMetricData.html).

|Quote from documentation|
|---|
|<img width="562" alt="image" src="https://user-images.githubusercontent.com/2962963/167301144-163343e7-5fb3-4aa4-bcf9-9cc6b421f098.png"> |

Not sure if it's critical at all, but maybe we can add another optional field called "unit" and let the user fill this information manually.

**Comment by or-shachar** [comment]
WDYT about removing `unit` from HELP altogether?

**Comment by matthiasr** [comment]
I'm fine with removing it if the data isn't available. The help string is nice but not essential.
